Algorithms/Introduction/fibonacci_iterative.rb

```diff
@@ -0,0 +1,19 @@
+def fib(n)
+  return nil if n.nil?
+
+  count = 1
+  fib_0 = 0
+  fib_1 = 1
+
+  if n == 0
+    return 0
+  elsif n <= 10
```

Comment:
While I understand why you put this, the original algorithm does not have it. Algorithms are typically designed to be followed to a T as far as process goes.